### DISCUSSION TITLE
HELP!

### DISCUSSION THREAD

**Discussion by CorruptedAngel7106** [community/community#110510] [GitHub Education] [answered]
### Hi everyone,

I am trying to get education plan for github but i do not have a student id yet since i just enrolled in a new school i tryed uploading my schedule for the school with my name and student id and a policy aggreement form as well but neither worked and i cannot add 2fa either because i am using a school computer witch does not allow me to download external applications

**Comment by samus-aran** [comment, marked as answer]
Hi @CorruptedAngel7106 👋 

We highly recommend you enable 2FA when applying for GitHub Education verification. We know 2FA may be a new process for some folks and with that comes questions. We put together a [FAQ](https://github.com/orgs/community/discussions/68240) that will help answer your concerns above. If you still have more questions, feel free to ask them here.
